content/js/getting-started.md

```diff
@@ -0,0 +1,46 @@
+<a name="sign-up-portal"></a>
+## Signing up at Portal
+
+To start using the Skygear JS SDK, you need to register your account and
+application at the [Skygear Developer Portal](https://portal.skygear.io/signup).
+You will be able to monitor your Skygear server and acquire information, such
+as your `Server EndPoint` and `API Key`, on the portal page.
+
+<a name="about-js-sdk"></a>
+## About the SDK
+
+Skygear provides a complete and open-source backend solution for your
+web applications. Our aim is to free front end developers from worrying about
+writing backend code or maintaining database.
```

Comment:
Still sound little weird. Perhaps re-phrase it a little bit so the paragraph starts with the JS SDK but not the server?